Gems/Multiplayer/Code/Source/Editor/MultiplayerEditorConnectionViewportDebugSystemComponent.cpp

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+
+#include "MultiplayerEditorConnectionViewportDebugSystemComponent.h"
+
+
+namespace Multiplayer
+{
+    void MultiplayerEditorConnectionViewportDebugSystemComponent::Reflect(AZ::ReflectContext* context)
+    {
+        if (AZ::SerializeContext* serializeContext = azrtti_cast<AZ::SerializeContext*>(context))
+        {
+            serializeContext->Class<MultiplayerEditorConnectionViewportDebugSystemComponent, AZ::Component>()
+                ->Version(1);
+        }
+    }
+
+    void MultiplayerEditorConnectionViewportDebugSystemComponent::Activate()
+    {
+        AzToolsFramework::EditorEvents::Bus::Handler::BusConnect();
+        MultiplayerEditorConnectionViewportDebugRequestBus::Handler::BusConnect();
+    }
+
+    void MultiplayerEditorConnectionViewportDebugSystemComponent::Deactivate()
+    {
+        AzToolsFramework::EditorEvents::Bus::Handler::BusDisconnect();
+        MultiplayerEditorConnectionViewportDebugRequestBus::Handler::BusDisconnect();
+    }
+
+    void MultiplayerEditorConnectionViewportDebugSystemComponent::NotifyRegisterViews()
+    {
+        IEditor* editor = nullptr;
+        AzToolsFramework::EditorRequests::Bus::BroadcastResult(editor, &AzToolsFramework::EditorRequests::GetEditor);
+        if (editor)
+        {
+            editor->RegisterNotifyListener(this);
+        }
+    }
+
+    void MultiplayerEditorConnectionViewportDebugSystemComponent::OnTick(float, AZ::ScriptTimePoint)
+    {
+        if (!m_debugText.empty())
+        {
+            constexpr int screenposition_x = 32;
```

Comment:
If these were cvars then they could be tuned at runtime or even customized by users with poor eyesight?